app-text/a2ps/a2ps-4.15.6.recipe

```diff
@@ -88,13 +91,15 @@ INSTALL()
 
 TEST()
 {
-	# TOTAL: 23
-	# PASS:  3
-	# SKIP:  0
-	# XFAIL: 0
-	# FAIL:  20
-	# XPASS: 0
-	# ERROR: 0
-	# mostly "a2ps: unknown medium `A4'"
+	# workaround for missing paperspecs config file in haikuporter chroot
+	# these are the paper formats that are used in tests
+	cat <<- EOF > $(finddir B_SYSTEM_SETTINGS_DIRECTORY)/paperspecs
+		A4,210,297,mm
+		Letter,8.5,11,in
+		A5,148,210,mm
+		A3,297,420,mm
+		Tabloid,11,17,in
```

Comment:
This is just the list of formats the tests for psutils use. I just added them one by one (copying them from the installed version of the file) and re-ran the tests until the test log didn't complain about a missing format. If a2ps tests e.g. only need A4, you could use an `echo` statement as above in BUILD, for example.

Reply:
Thanks on the pointer, the line from BUILD() was enough to pass the test, 1 is failing, but I get no information on that failing test.